apps/web/lib/surveys/surveys.ts

```diff
@@ -456,3 +459,27 @@ function matchAndUpdateArray(choices: any, responseValue: string[]) {
 
   return responseValue;
 }
+
+// generate encrypted single use id for the survey
+export const generateSurveySingleUseId = (isEncrypted: boolean): string => {
+  const cuid = cuid2.createId();
+  if (!isEncrypted) {
+    return cuid;
+  }
+  const encryptedCuid = encryptAES128(env.FORMBRICKS_ENCRYPTION_KEY!, cuid);
```

Comment:
Please move `FORMBRICKS_ENCRYPTION_KEY` to constants.ts and import from there. This way we can make sure that a build error is thrown when imported into a client component. Just to be sure ;-)